src/translations/fr/translation.json

```
@@ -1,300 +1,300 @@
-    "Best": "Best",
-    "Empty fuel": "Empty fuel",
-    "12.12 KEYS - MARKET PRICING HISTORY": "12.12 KEYS - MARKET PRICING HISTORY",
-    "Labs": "Labs",
-    "Interchange": "Interchange",
-    "Shoreline": "Shoreline",
-    "Reserve": "Reserve",
-    "Graphs made by": "Graphs made by",
+    "Is it free?": "C'est gratuit?",
+    "Yes. It does however cost money for us to run and maintain, so please consider supporting us on": "Oui. Cependant, cela coûte de l'argent pour nous de fonctionner et de maintenir, alors s'il vous plaît envisager de nous soutenir sur",
+    "Is there a rate limit?": "Y a-t-il une limite de taux?",
+    "Wait... really?": "Attendez... vraiment ?",
+    "Yeah. It's actually better if you": "Oui. C'est en fait mieux si vous",
+    "don't": "ne le faites pas",
+    "limit your requests to get as fresh data as possible": "limitez votre demande pour obtenir de nouvelles données autant que possible",
+    "Where is the data from?": "D'où viennent les données ?",
+    "We source data from multiple places to build an API as complete as possible. We use data from:": "Nous recueillons des données à partir de plusieurs endroits pour créer une API aussi complète que possible. Nous utilisons les données de :",
+    "Directly from the game": "En direct du jeu",
+    "Our network of scanners": "Notre réseau de scanners",
+    "Examples": "Exemples",
+    "example": "Exemple",
+    "Tarkov Tools API Users": "Utilisateurs de l'API Tarkov Tools",
+    "Want to be included on this page? Join the": "Voulez-vous être inclus sur cette page? Rejoignez le",
+    "and tell us about what you've made!": "et racontez-nous ce que vous avez fait !",
+    "Barter Profits": "Bénéfices du troc",
+    "Show all barters": "Afficher tous les trocs",
+    "Item filter": "Filtre d'articles",
+    "filter on item": "filtrer sur l'article",
+    "All": "Tout",
+    "Hide dogtags": "Cacher les plaques d'identification",
+    "Connected to": "Connecté à",
+    "id to control": "identifiant pour contrôler",
+    "Remote Control": "Télécommande",
+    "View map": "Voir la carte",
+    "Go": "Aller",
+    "View caliber": "Voir le calibre",
+    "Load tarkov.dev in another browser or window to control it from here": "Chargez tarkov.dev dans un autre navigateur ou une autre fenêtre pour le contrôler à partir d'ici",
+    "Hideout Crafts": "Artisanat de cachette",
+    "Most profitable craft in each station": "L'embarcation la plus rentable dans chaque station",
+    "Sets all fuel prices to 0, as that's more or less the case if you use fuel to power your hideout": "Définit tous les prix du carburant à 0, comme c'est plus ou moins le cas si vous utilisez du carburant pour alimenter votre cachette",
+    "Best": "Meilleur",
+    "Empty fuel": "Carburant vide",
+    "12.12 KEYS - MARKET PRICING HISTORY": "12.12 CLÉS - HISTORIQUE DES PRIX DU MARCHÉY",
+    "Labs": "Laboratoires",
```

```suggestion
    "Labs": "Labo",
```

---

```suggestion
    "Labs": "Labos",
```

Can we use Labos instead ? As Labos is labs (plural) whereas labo is just lab